### PR TITLE
[Site] GitOps Page

### PR DESCRIPTION
**Pull request by Udit-takkar**
**Description**

This PR fixes #

https://github.com/layer5io/layer5/pull/4300

TODO

- [x] Responsive bugs
- [ ] Animate hero icons

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit e42af4be3952441ca40eb450edb025157abfb4b7 at: https://6480ec9fc17e9e3fcd0446c7--layer5.netlify.app

**Comment by Udit-takkar** [comment]
@leecalcote okay i thought you needed that commits. i will remove these later

**Comment by vishalvivekm** [comment]
@Udit-takkar Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. :)

**Comment by Chadha93** [comment]
:eyes:

**Comment by leecalcote** [comment]
@Udit-takkar this still has the Platform Engineering blog post in it. Please remove immediately, so that we can proceed.

**Comment by l5io** [comment]
🚀 Preview for commit 4f14bd4e2b21bbf10744c87e766ce5d5b7b56015 at: https://648b5a38736244348f6d49c1--layer5.netlify.app

**Comment by Chadha93** [comment]
We don't have an entry for this in the mega menu yet, what type of animation do we want here, similar to the Design page for "Design your infrastructure"? @Shivam-AfA will you toss in the responsiveness and animation update?

**Comment by Shivam-AfA** [comment]
Sure @Chadha93

**Comment by l5io** [comment]
🚀 Preview for commit afdb3e8815d7b8ea4e51b992fa5d8deb764c9ce5 at: https://648cc9e2b32a7910fc46655f--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 5a858f906311d4a53b51a4decd68ca55e7d7d148 at: https://648cd43238df29191a1521c6--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 1cff6eab9fadd361f21237f67223d2d8387be552 at: https://648d8063b32a796b32466510--layer5.netlify.app

File: src/sections/gitops/index.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+      <div className="content-container content-container-more-width">
+        <Row className="content-row">
+          <Col lg={6} md={6} className="desc-text-container">
+            <h1 className="heading-1">MeshMap GitHub Action</h1>
+            <SquarePoint>See your deployment before you merge</SquarePoint>
+            <SquarePoint>Connect MeshMap to your GitHub repo and see changes pull request-to-pull request</SquarePoint>
+            <SquarePoint>Get snapshots of your infrastructure directly in your PRs</SquarePoint>
+            <div style={{ display: "flex", justifyContent: "center" }}>
+              <Button primary className="learn-more-btn" title="Install" url="./operating-service-meshes"/>
```
Comment (leecalcote) [inline src/sections/gitops/index.js:53]:
```suggestion
              <Button primary className="learn-more-btn" title="Install" url="https://cloud.layer5.io/connect/github/new"/>
```

File: src/components/Call-To-Actions/CTA_FullWidth/index.js [hunk 2]
```diff
@@ -94,7 +94,7 @@ const CTA_FullWidthWrapper = styled.div`
 `;
 
 const defaultHeading = "Layer5 Community";
-const defaultContent = "Join the Layer5 community and explore the world of service meshes!";
+const defaultContent = "Join the Layer5 community and explore the world of cloud native!";
```
Comment (leecalcote) [inline src/components/Call-To-Actions/CTA_FullWidth/index.js:97]: Good eye. 👍

**Comment by l5io** [comment]
🚀 Preview for commit b2f97d02bd40802235ec4f578897e7f540d812eb at: https://648e11c673a33f3e4fc9dedf--layer5.netlify.app

**Comment by Chadha93** [comment]
Merging this, only the animation part is left.

**Pull request merged**